api/src/pcapi/routes/public/individual_offers/v1/events.py

```diff
@@ -497,6 +497,8 @@ def post_event_stocks(event_id: int, body: serialization.EventStocksCreation) ->
                         id_at_provider=date.id_at_provider,
                     )
                 )
+                if existing_stocks_count == 0:
```

Comment:
j'ai un doute, que ce passe-t-il si il n'y a aucun stock dans body.dates ? 
Si on a une 400 niquel, sinon je pense que la condition doit être 
`if not existing_stocks_count and body.dates:`